csharp_dotnetcore/51.Enterprise-Bot-Template/EnterpriseBot.cs

```diff
@@ -14,29 +15,27 @@ namespace EnterpriseBot
     /// </summary>
     public class EnterpriseBot : IBot
     {
-        private readonly BotServices _services;
-        private readonly SemaphoreSlim _semaphore;
-        private readonly EnterpriseBotAccessors _accessors;
+        private BotServices _services;
+        private ConversationState _conversationState;
+        private UserState _userState;
```

Comment:
Why were the readonly attributes removed? these look like they should be readonly.